RockLib.Compression/RockLib.Compression.csproj

```diff
@@ -17,6 +17,7 @@
     <PublishRepositoryUrl>True</PublishRepositoryUrl>
     <EmbedUntrackedSources>True</EmbedUntrackedSources>
     <DebugType>Embedded</DebugType>
+    <TargetFrameworks>net6.0;net8.0;net48</TargetFrameworks>
```

Comment:
I think this should be updated in `Directory.Build.props`....while you're in there, you'll want to change some of the `PropertyGroup` conditions.